lms/djangoapps/certificates/models.py

```diff
@@ -436,6 +436,23 @@ def get_certificate_invalidations(cls, course_key, student=None):
             })
         return data
 
+    @classmethod
+    def is_certificate_invalid(cls, course_key, student):
+        """Check that certificate for a student is invalidated or not.
+
+        Arguments:
+            course_key (CourseKey): The course associated with the certificate.
+            student (user): logged-in user
+
+        Returns:
+             Boolean denoting whether certificate is invalid or valid.
```

Comment:
`Boolean denoting whether the student in the course is invalidated to receive certificates`